app/src/androidTest/java/com/github/polypoly/app/commons/PolyPolyTest.kt

```diff
@@ -70,27 +69,27 @@ abstract class PolyPolyTest(
 
         val TEST_GAME_LOBBY_FULL = GameLobby(
             TEST_USER_0, GameParameters(GameMode.RICHEST_PLAYER, 4, 6,
-            60, 20, getZones(), 100), "Full gameLobby", "lobby1234"
+            60, 20, getZones(), 100), "Full gameLobby", "11111"
```

Comment:
Why did these tests change in this PR? 🤨

Reply:
because I finally used the constant for loby code length in the field. So the tests failed because these codes were larger than 5